frontend/src/Draw.tsx

```diff
@@ -92,7 +93,7 @@ class Draw extends React.Component<IDrawProps, IDrawState> {
     ) {
       this.timeout = this.props.setTimeout(() => {
         this.drawCard();
-      }, 250);
+      }, this.props.autodrawSpeedMs || 250);
```

Comment:
while this is valid JS the typescript compiler doesn’t love it — consider instead

`this.props.autodrawSpeedMs !== null ? this.props.autodrawSpeedMs : 250`